include/d/d_bg_s_roof_chk.h

```diff
@@ -21,7 +21,7 @@ class dBgS_RoofChk : public cBgS_PolyInfo, public cBgS_Chk, public dBgS_Chk {
 
     virtual ~dBgS_RoofChk() {}
 
-    void SetPos(cXyz& pos) { m_pos = pos; }
+    void SetPos(cXyz const& pos) { m_pos = pos; }
```

Comment:
Careful copy pasting inlines from TP. The devs changed tons of functions to be const after TWW's release for TP, so many of them that should be const in TP should not be for TWW. And these copy-pasted consts actually affect regalloc and cause other matching problems.